packages/bundlers/experimental/src/ExperimentalBundler.js

```diff
@@ -670,8 +672,9 @@ function createIdealGraph(
             );
             if (
               bundle !== 'root' &&
-              bundle.bundleBehavior !== 'inline' &&
-              !bundle.env.isIsolated()
+              bundle.bundleBehavior == null &&
```

Comment:
This line is what caused the test change, where a bundle was previously internalized and now isn't. I think in the general case, however, that isolated bundles should not be reachable.